cmd/notification.go

```diff
@@ -1044,6 +1044,35 @@ func (sys *NotificationSys) DispatchNetOBDInfo(ctx context.Context) []madmin.Ser
 	return serverNetOBDs
 }
 
+// DispatchNetOBDChan - Net OBD information from other nodes
+func (sys *NotificationSys) DispatchNetOBDChan(ctx context.Context) chan madmin.ServerNetOBDInfo {
+	serverNetOBDs := make(chan madmin.ServerNetOBDInfo)
+	wg := sync.WaitGroup{}
+
+	wg.Add(1)
```

Comment:
shouldn't this be moved to line 1057 so each peer can be added to wait group?

Reply:
no, this dispatching is done serially. The wg is needed to close the channel once the netobd tests are complete